highlight/highlight
# Remove query builder

## Summary
Swap out the query builder in the project filters, and clean up any dead code associated with the query builder

https://www.loom.com/share/c9921c6005a345dba1519c2710274f59

## How did you test this change?
1) On `main`, update workspace to be in the "enterprise" tier: `UPDATE workspaces SET plan_tier = 'Enterprise' WHERE id = 1;`
2) Click into project settings and add project filters for errors and sessions
3) Check the query in Postgres
4) Switch to this branch
5) Load the project filters created with the old query builder
- [ ] Able to load filters
- [ ] Able to edit filters
- [ ] Format matched in postgres

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
